src/reblox/pseudo_random.rs

```diff
@@ -0,0 +1,16 @@
+
+pub struct PseudoRandom {
+    pub seed: u64
+}
+
+impl PseudoRandom {
+    pub fn new() -> PseudoRandom {
+        PseudoRandom { seed: 0 }
+    }
+
+    pub fn next(&mut self) -> u64 {
```

Comment:
Perhaps use the PRNG provided by libextra (see `minesweeper`).

On a more technical note: RNGs generally applies some bijective function to the result to avoid patterns (e.g. XOR with some number, wrapping multiplication with a prime, wrapping addition, and so on).